lib/rules/focus-order-semantics.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "focus-order-semantics",
+  "selector": "div, h1, h2, h3, h4, h5, h6, [role=heading], p, span",
+  "matches": "inserted-into-focus-order-matches.js",
+  "tags": ["cat.keyboard", "best-practice", "experimental"],
+  "metadata": {
+    "description": "Ensures elements placed in the focus order have an appropriate aria role for interactive content",
+    "help": "Elements placed in the focus order must have an aria role appropriate for interactive content."
```

Comment:
We need to shorten this so that it fits in the Axe extension properly. I suggest: "Focusable elements have an appropriate role" (no "." at the end either please).

Reply:
Shortened, though I'm still trying to avoid equating focusable with in-focus-order so I didn't change it quite as you recommend. It's shorter than some other descriptions at least (e.g [link-in-text-block](https://github.com/dequelabs/axe-core/blob/develop/lib/rules/link-in-text-block.json#L141))